content/guides/docker-concepts/getting-started/get-docker.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Get Docker 
+keywords: concepts, container, docker desktop
+description: This concept page will teach you download Docker Desktop and install it on Windows, Mac, and Linux
+---
+
+{{< youtube-embed C2bPVhiNU-0 >}}
+
+## Explanation
+
+Docker Desktop is the all-in-one package to build images, run containers, and so much more.
+This guide will walk you through the installation process, enabling you to experience Docker Desktop firsthand.
+
+
+> **Docker Desktop terms**
+>
+> Commercial use of Docker Desktop in larger enterprises (more than 250
+> employees OR more than $10 million USD in annual revenue) requires a [paid subscription](https://www.docker.com/pricing/?_gl=1*1nyypal*_ga*MTYxMTUxMzkzOS4xNjgzNTM0MTcw*_ga_XJWPQMJYHQ*MTcxNjk4MzU4Mi4xMjE2LjEuMTcxNjk4MzkzNS4xNy4wLjA.).
+
+{{< card
+  title="Docker Desktop for Mac"
+  description="[Download (Apple Silicon)](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64) | [Download (Intel)](https://desktop.docker.com/mac/main/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64) | [Additional setup instructions](/desktop/install/mac-install)"
```

Comment:
We haven't shared any install instructions on this pages. Should the "Additional setup instructions" be updated to Install instructions similar to the DD for Linux section below?

Reply:
You're right. I just updated "Additional setup instructions" to "Install instructions" for Mac and Windows.